orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/v1schema/render/JinjaRenderer.java

```diff
@@ -67,6 +69,10 @@ public JinjaRenderer(RenderedValueConverter renderedValueConverter, ObjectMapper
     jinja.setResourceLocator(new NoopResourceLocator());
     jinja.getGlobalContext().registerTag(new ModuleTag(this, pipelineTemplateObjectMapper));
```

Comment:
Opted to leave these tags as defaults. A little bit of smell with `ModuleTag` as it has a circular dependency with the renderer itself. That made it hard to extract into its own component without having to do some Spring handwaving